src/components/Header.js

```diff
@@ -53,22 +54,54 @@ const Header = class extends React.Component {
 
   if (isMobile) {
 
-    let nM = document.getElementById("navMenu");
-    nM.style.backgroundColor = "#abd6d1"
-    // Get all "accordions" elements
-    const accordion = Array.prototype.slice.call(document.querySelectorAll('.accordions'), 0);
-    // Check if there are any navbar links
-    if (accordion.length > 0) {
+    // Return an array of bulmaCollapsible instances (empty if no DOM node found)
+   const bulmaCollapsibleInstances = bulmaCollapsible.attach('.is-collapsible');
```

Comment:
@CreativeBulma For a minimal configuration, does it is enough to open and close an accordion?

Reply:
Yes it's the only code you need for basic behavior.
If you don't plan to manipulate instances later you don't even need to save them into a const.
If you want an element to be opened by default just add <code>is-active</code> class on it.

Reply:
@CreativeBulma thank you, i think i will use the simplest one. Anyway i will do other tests.